cmd/ocm/create/cluster/cmd.go

```diff
@@ -514,7 +527,20 @@ func preRun(cmd *cobra.Command, argv []string) error {
 		parseSubscriptionType(args.subscriptionType) == parseSubscriptionType(gcpSubscriptionTypeTemplate.Value)
 
 	if isGcpMarketplace {
-		fmt.Println("setting provider to", c.ProviderGCP)
+		if args.interactive {
+			fmt.Println("setting provider to", c.ProviderGCP)
+			fmt.Println("setting ccs to 'true'")
```

Comment:
this change the behaviour: why do we only show the message/override the ccs and cloud provider attributes in the interactive mode?

Reply:
Added the prompt messages for non-interactive mode as well